mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -39,8 +39,9 @@
 # existing clients.
 _PROTOCOL_MINOR_VERSION = 0
 
-_LAUNCH_CMD = ('%s am instrument -w -e action start %s/' +
-               _INSTRUMENTATION_RUNNER_PACKAGE)
+_LAUNCH_CMD = (
+    '{shell_cmd} am instrument --user {user} -w -e action start {snippet_package}/'
```

Comment:
note: current user is the default for `am instrument`. However, personally I still think it's better practice to be explicit (so no recommended change)